### PR TITLE
Menu qt

### PR DESCRIPTION
**Pull request by oneeyeman1**
Implement {Set,Get}Bitmap for menuitem on Qt

Just 2 files are modified - include/wx/qt/menuitem.h and src/qt/menuitem.cpp

Thank you for review.

Please check and apply.

### CODE REVIEW

**Comment by MaartenBent** [comment]
There are some changes to the `src/osx/cocoa/textctrl.mm` and the `text` sample as well, are these intended?

**Comment by oneeyeman1** [comment]
@vadz ,
Is it better to grab latest sources and resubmit the PR or you can grab the patch from the ticket and apply? Or you can just apply the commit?

Let me know...

Thank you.

**Comment by vadz** [comment]
It's better to squash everything together and force push. I can't "grab" anything as it conflicts with the changes in master and also fails the CI build...

**Comment by oneeyeman1** [comment]
@vadz ,
OK, I will get the latest sources and re-create the PR tonight.
Or you can get the patch from the ticket I made and apply it. But in any case I will try to make something tonight.

Thank you.

**Review by MaartenBent — COMMENTED** [review]
Tested it and the `ClearLog` icon in the `menu` sample is now shown.

This PR still contains unrelated changes to `textctrl`.

**Comment by oneeyeman1** [comment]
@MaartenBent ,
Thanx for testing.
I'm planning to resubmit this PR based on the instructions I received on wx-dev ML.
Probably either today or tomorrow it will be a new pull request which will be generated properly.

I will continue to work on different pieces of the wxQt and submit the PR based on my changes.
I just prefer to keep this one until the new one will be submitted so that the code can be moved.

Thank you again.

File: src/qt/menuitem.cpp [hunk 1]
```diff
@@ -106,18 +106,17 @@ bool wxMenuItem::IsChecked() const
...
-{
-    wxMISSING_FUNCTION();
-
-    static wxBitmap s_bitmap;
-
-    return s_bitmap;
+    if( m_kind == wxITEM_NORMAL )
+    {
+        m_bitmap = bitmap;
+        m_qtAction->setIcon( QIcon( *bitmap.GetHandle() ) );
```
[the remark this answers is not in the document]
Comment (oneeyeman1) [reply src/qt/menuitem.cpp]: I will check what is being used by other ports.

**Comment by vadz** [comment]
Replaced with #975.

**Pull request closed without merging**